src/test/java/harness/CRLJanitor.java

```diff
@@ -101,17 +102,16 @@ public static void registerWorkspaceForCleanup(UUID uuid, TestUser testUser) {
       return;
     }
     String wsmInstance = Context.getServer().getWorkspaceManagerUri();
+    OffsetDateTime curOffsetDateTime = OffsetDateTime.now().truncatedTo(ChronoUnit.MILLIS);
     CreateResourceRequestBody janitorRequest =
         new CreateResourceRequestBody()
             .resourceUid(
                 new CloudResourceUid()
-                    .terraWorkspace(
-                        new TerraWorkspaceUid()
-                            .workspaceId(uuid)
-                            .workspaceManagerInstance(wsmInstance)))
-            .resourceMetadata(new ResourceMetadata().workspaceOwner(testUser.email))
-            .creation(OffsetDateTime.now())
-            .expiration(OffsetDateTime.now().plus(WORKSPACE_TIME_TO_LIVE));
+                    .googleProjectUid(
```

Comment:
can we please split this to a different PR? doesn't seem related to jdk upgrade.

Reply:
This is actually required due to a breaking change when the library versions were upgraded (to keep up with Java 17 upgrade).